.github/workflows/publish.yml

```
@@ -0,0 +1,47 @@
+name: publish
+'on':
+  push:
+    branches:
+      ktp-docs-ci
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: install dependencies
+      run: |
+        sudo apt update
+        sudo apt install -y pandoc git

```

```suggestion
        sudo apt install -y pandoc

```
How come git is necessary?